pydarn/plotting/superdarn_radars.py

```diff
@@ -1,101 +1,116 @@
-# Copyright (C) 2019 SuperDARN
-# Author: Marina Schmidt
+# Copyright (C) SuperDARN Canada, University of Saskatchewan
+# Authors: Marina Schmidt and Danno Peters
 
 """
 This module contains SuperDARN radar information
 """
 from typing import NamedTuple
 from enum import Enum
+from datetime import datetime
```

Comment:
Why not `import datetime as dt`?  Leads to a bit less confusion when reading the code.

Reply:
Looking at other code, it seems that it is standard in this module to import datetime from datetime, so I am resolving this.